packages/action-password-update/src/components/PasswordUpdateDialog/PasswordUpdateDialog.scss

```diff
@@ -1,54 +1,206 @@
-.PasswordUpdateDialog {
+/*****
+* import sass dependencies
+*/
 
-  .oldPassword {
-    margin-bottom: 1.5em;
-  }
+// TODO reduce redundancy
+// TODO get rid of cumbersome imports
+// TODO make it possible to use _variables.scss from customer
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_opacity.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_vendor-prefixes.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_background-variant.scss";
+@import "../../../../../node_modules/font-awesome/scss/_variables.scss";
 
-  .ValidationRules {
-    margin-bottom: 1.5em;
-  }
+$fa-font-path: "../../../../../node_modules/font-awesome/fonts/";
+@import "../../../../../node_modules/font-awesome/scss/font-awesome.scss";
+
+
+/*****
+* tocco-client boilerplate
+* for styling convenience while developing, only used inside monorepo tocco-client
+*/
+
+// TODO replace #root with #tocco-client-development
+#root {
+  margin: 50px;
+  border: 1px dashed $gray-lighter;
+}
+
+
+/*****
+* extend bootstrap
+*/
+
+// control whitespace rhythm
+.form-group {
+  margin-bottom: $line-height-computed;
+}
+
+// list styling
+ul {
+  margin-bottom: $line-height-computed;
+}
+li > ul {
+  margin-bottom: 0;
+}
+
+
+/*****
+/* coperate design variables
+*/
+
+// TODO cetralize coperate design variables
+$icon-list-item-1: '\2022';  /* bullet point */
+$icon-valid: $fa-var-check;
+$icon-invalid: $fa-var-times;
+
+
+/*****
+* icon list
+* prefix list items by any character
+*/
 
-  .PasswordMatchDisplay {
-    margin-bottom: 1.5em;
+$icon-list-icon-size: floor($font-size-base * 1.5);
+$icon-list-indent: $icon-list-icon-size;
+
+// base style
+.icon-list {
+  list-style-type: none;
+  padding-left: $icon-list-indent;
+
+  li {
+    position: relative;
+
+    &:before {
+      font-family: 'FontAwesome';
+      position: absolute;
+      left: -$icon-list-indent;
+      content: $icon-list-item-1;
+    }
   }
+}
 
-  .FailureMessage {
-    margin-top: 1em;
+// replace icon for list items with a special class
```

Comment:
This is meant as information for developers to understand concealed purpose of a declaration or property. Every comment which starts with TODO is a open task. All other comments remain until the end of time. Would you like to indicate informations by INFO or something like that.